.gitignore

```diff
@@ -40,4 +40,4 @@ yarn-debug.log*
 # Ignore Mac and Linux file system files
 *.swp
 .DS_Store
-.env*
```

Comment:
Ei @tsainei , could you explain why you have deleted this file? It is not clear to me.

Thank youuuu <3